integreat_cms/api/v3/chat/zammad_api.py

```diff
@@ -90,23 +93,26 @@ def _parse_response(self, response: dict | list[dict]) -> dict | list[dict]:
             return [self._parse_response(item) for item in response]  # type: ignore[misc]
 
         if author := response.get("sender"):
-            response["user_is_author"] = author == "Customer"
+            response["user_is_author"] = (
+                author == "Customer" and response["subject"] != AUTO_ANSWER_STRING
+            )
+        # response["automatic_answer"] = response["subject"] == AUTO_ANSWER_STRING # TODO: Why does this not work?
```

Comment:
We get a key error here. That probably means that the subject attribute does not exist.